src/main/java/fact/utils/ConvertSinglePulses2Timeseries.java

```diff
@@ -0,0 +1,90 @@
+package fact.utils;
+
+import fact.Utils;
+import fact.features.singlePulse.timeSeriesExtraction.AddFirstArrayToSecondArray;
+import fact.features.singlePulse.timeSeriesExtraction.SinglePulseExtractor;
+import fact.features.singlePulse.timeSeriesExtraction.TemplatePulse;
+import fact.features.singlePulse.timeSeriesExtraction.ElementWise;
+import org.apache.commons.lang3.ArrayUtils;
+import org.apache.commons.math3.analysis.function.Sin;
+import stream.Data;
+import stream.Processor;
+import stream.annotations.Parameter;
+
+import javax.rmi.CORBA.Util;
```

Comment:
I don't know what this is. Probably an accidental import right? sometimes intellij is too fast for me ^^